drivers/staging/vc04_services/interface/vchiq_arm/vchiq_core.c

```diff
@@ -536,6 +537,7 @@ remote_event_wait(wait_queue_head_t *wq, struct remote_event *event)
 	return 1;
 }
 
+/* FIXME */
```

Comment:
Acknowledge that the event has been signalled, and wake any waiters. Usually called as a result of the doorbell being rung.